kotlin/surface-placement/src/main/java/com/esri/arcgisruntime/sample/surfaceplacement/MainActivity.kt

```diff
@@ -0,0 +1,204 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.esri.arcgisruntime.sample.surfaceplacement
+
+import android.graphics.Color
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.geometry.Point
+import com.esri.arcgisruntime.layers.ArcGISSceneLayer
+import com.esri.arcgisruntime.mapping.ArcGISScene
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.view.Camera
+import com.esri.arcgisruntime.mapping.view.Graphic
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay
+import com.esri.arcgisruntime.mapping.view.LayerSceneProperties.SurfacePlacement
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol
+import com.esri.arcgisruntime.symbology.TextSymbol
+import com.esri.arcgisruntime.symbology.TextSymbol.VerticalAlignment
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    // create a scene
+    val scene = ArcGISScene().apply {
+      // add a basemap to it
+      basemap = Basemap.createImagery()
+      // add base surface for elevation data
+      baseSurface.elevationSources.add(
+        ArcGISTiledElevationSource(getString(R.string.elevation_image_service))
+      )
+      // create a scene layer from the Brest, France scene server
+      operationalLayers.add(
+        ArcGISSceneLayer(getString(R.string.scene_layer_service))
+      )
+    }
+
+    // set an initial viewpoint
+    val initialViewPoint =
+      Point(-4.45968, 48.3889, 100.9922)
+    val camera =
+      Camera(initialViewPoint, 329.91, 80.0, 0.0)
+
+    sceneView.apply {
+      this.scene = scene
+      setViewpointCamera(camera)
+    }
+
+    // create point for the scene related graphic with a z value of 0
+    val sceneRelatedPoint =
+      Point(
+        -4.4610562,
+        48.3902727,
+        0.0,
+        camera.location.spatialReference
+      )
+    // create point for the surface related graphics with z value of 70
+    val surfaceRelatedPoint =
+      Point(
+        -4.4609257,
+        48.3903965,
+        70.0,
+        camera.location.spatialReference
+      )
+
+    // create a red triangle symbol
+    val triangleSymbol =
+      SimpleMarkerSymbol(SimpleMarkerSymbol.Style.TRIANGLE, Color.RED, 10f)
+
+    // create a text symbol for elevation mode
+    val drapedFlatText = TextSymbol(
+      15F, "DRAPED FLAT", Color.BLUE, TextSymbol.HorizontalAlignment.LEFT,
+      VerticalAlignment.TOP
+    ).apply { offsetY = 20f }
```

Comment:
```suggestion
    ).apply { 
    offsetY = 20f 
    }
```

This is slightly more readable to me, but I accept this might just be a personal preference :)

Reply:
I went back and forth on this several times, and now I can't tell anymore, so I'll do as you say and apply it to all of them